contracts/exchange-wrappers/ZeroExExchangeWrapper.sol

```diff
@@ -0,0 +1,50 @@
+/*
+    Copyright 2021 dYdX Trading Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity ^0.8.0;
+
+
+import { I_ExchangeWrapper } from "../external/I_ExchangeWrapper.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ZeroExExchangeWrapper is I_ExchangeWrapper  {
```

Comment:
Nit: If this contract is just for tests then put `Mock` in the contract name

Reply:
Going to hold off on doing that because this contract _wont_ eventually be just a mock